apps/asap-server/src/handlers/jobs/resubscribe-calendars.ts

```diff
@@ -0,0 +1,64 @@
+import { DateTime, Duration } from 'luxon';
+import Calendars, { CalendarController } from '../../controllers/calendars';
+import {
+  UnsubscribeFromEventChanges,
+  SubscribeToEventChanges,
+  subscribeToEventChangesFactory,
+  unsubscribeFromEventChangesFactory,
+} from '../webhooks/webhook-calendar-created';
+import getJWTCredentials from '../../utils/aws-secret-manager';
+import logger from '../../utils/logger';
+import { ScheduledHandlerAsync } from '../../utils/types';
+
+export const resubscribeCalendarsHandlerFactory = (
+  calendarController: CalendarController,
+  unsubscribe: UnsubscribeFromEventChanges,
+  subscribe: SubscribeToEventChanges,
+): ScheduledHandlerAsync => async () => {
+  const now = DateTime.local();
+  const duration24h = Duration.fromObject({ day: 1 });
+  const calendars = await calendarController.fetchRaw({
+    maxExpiration: now.plus(duration24h).toMillis(),
+    take: 100,
```

Comment:
FYI i dont think we expect a 100 calendars to all expire at the same time and if they do - we will resubscribe only the first 100 then on the next day - the remaining ones